src/sourcekit-lsp/LanguageClientManager.ts

```diff
@@ -209,17 +211,23 @@ export class LanguageClientManager {
      * @returns result of process
      */
     async useLanguageClient<Return>(process: {
-        (client: langclient.LanguageClient, cancellationToken: vscode.CancellationToken): Return;
-    }) {
-        if (!this.languageClient) {
+        (
```

Comment:
I cleaned up `useLanguageClient` a little because I was originally using this in the PR, but I'm not anymore. But the clean up is still valid I feel.